recipe/meta.yaml

```
@@ -1,12 +1,12 @@
-{% set version = "3.25.0" %}
+{% set version = "3.25.1" %}
 
 package:
   name: conda-smithy
   version: {{ version }}
 
 source:
   url: https://github.com/conda-forge/conda-smithy/releases/download/v{{ version }}/conda-smithy-{{ version }}.tar.gz
```

I think this would be simpler. That way, we also don't have to specially create a "release" (in GH terminology), a git tag would be enough.
```suggestion
  url: https://github.com/conda-forge/conda-smithy/archive/refs/tags/v{{ version }}.tar.gz
```

---

Ah I see what is happening. When we make the release, we upload an artifact as part of the release

The reason we do this is GitHub archives are dynamically generated and can change due to process changes on the GitHub side. These result in changes in the checksum

Already had downloaded GitHub's artifact to generate the checksum. Have attached that same artifact to the release. So this should be fixed

Will restart CI

---

Looking at [the rever logic]( https://github.com/conda-forge/conda-smithy/blob/8a8ac2ee9a5e7d01e240bcdce56a44132f2d1cfe/rever.xsh ), think we would want to [use an `sdist`]( https://github.com/conda-forge/conda-smithy/blob/8a8ac2ee9a5e7d01e240bcdce56a44132f2d1cfe/rever.xsh#L12-L16 ), but trying to generate that locally either tried to do other steps that were already done (tagging, releasing, etc.) or just running `python setup.py sdist` resulted in an unversioned `sdist`. Probably missing something here